inst/builtin/bs5/shiny/_rules.scss

```
@@ -125,10 +125,11 @@ $bslib-checkbox-radio-margin-right: 0.35em !default;
   .bslib-page-sidebar {
     --bslib-page-sidebar-title-bg: transparent;
     --bslib-page-sidebar-title-color: var(--#{$prefix}body-color);
-
-    > .bslib-sidebar-layout > .main {
-      background-color: var(--bslib-shiny-preset-main-bg);
-      @include bslib-box-shadows;
+    > .bslib-sidebar-layout {
+      --bslib-sidebar-main-bg: var(--bslib-shiny-preset-main-bg);
```

I think this should use the private properties instead, to avoid cascading the value.

```suggestion
      --_main-bg: var(--bslib-shiny-preset-main-bg);
```

In fact, we should probably take another look at the dashboard background feature to make sure we're using the private properties wherever needed (e.g. there are other instances not in this diff that should be reviewed).

---

Oh wait, I just realized the cascading is probably intentional... let's talk through this in person.

---

Just added https://github.com/rstudio/bslib/pull/890/commits/0569bdbd240b1ff12254209d5263006d3c9d5aa0 to go the other way: we'll set `--bslib-dashboard-*` variables on `:root` in the shiny preset so they can be overridden globally. We do intend for them to cascade to all sidebar layouts, without which the styles clash with the main content area's background color.